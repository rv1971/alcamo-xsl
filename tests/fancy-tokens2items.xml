<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <xsl:stylesheet
      xmlns="http://www.w3.org/1999/xhtml"
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xml:id="xslt"
      version="1.0">
    <xsl:import href="../xsl/fancy-html.xsl"/>

    <xsl:template match="/">
      <a:result>
        <xsl:apply-templates select="/a:doc/a:data"/>
     </a:result>
    </xsl:template>

    <xsl:template match="a:item">
      <ul>
        <p>
          <xsl:value-of select="."/>
        </p>
        <xsl:apply-templates select="." mode="a:tokens2items"/>
      </ul>
    </xsl:template>
  </xsl:stylesheet>

  <a:data>
    <a:item>foo</a:item>
    <a:item>bar   baz QUX</a:item>
    <a:item>quux
corge</a:item>
  </a:data>
</a:doc>
