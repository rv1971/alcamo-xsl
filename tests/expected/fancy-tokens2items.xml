<?xml version="1.0"?>
<a:result xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#" xmlns="http://www.w3.org/1999/xhtml">
  <ul>
    <p>foo</p>
    <li>foo</li>
  </ul>
  <ul>
    <p>bar   baz QUX</p>
    <li>bar</li>
    <li>baz</li>
    <li>QUX</li>
  </ul>
  <ul>
    <p>quux
corge</p>
    <li>quux</li>
    <li>corge</li>
  </ul>
</a:result>
